tests/testthat/test_c_create_package.R

```diff
@@ -23,6 +23,13 @@ test_that("create_package() works", {
     regexp = sprintf("package created at `.*%s`", package)
   )
 
+  repo <- file.path(path, package)
```

Comment:
`create_package()` should initialise a git repo. So no need to do that manually.

Reply:
Done in 5c98cd1